compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/GenerateClosuresForDefaultValues.java

```diff
@@ -483,13 +495,22 @@ public void visit(BLangFunctionTypeNode functionTypeNode) {
 
     @Override
     public void visit(BLangSimpleVariable varNode) {
+        BLangExpression bLangExpression;
```

Comment:
Why do we need this here if we are returning in the if block?

Reply:
Removed

Reply:
Are your changes pushed?

Reply:
Yes, you mean the `bLangExpression variable`, right? I have updated the logic for that.
https://github.com/ballerina-platform/ballerina-lang/blob/3abc989bdd82a9a7d91a41d4a446c5efa1e9ed5c/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/ClosureGenerator.java#L525